Mail/Views/Thread/ThreadView.swift

```diff
@@ -95,7 +95,7 @@ struct ThreadView: View {
                 .padding(.bottom, value: .regular)
                 .padding(.horizontal, value: .regular)
 
-                MessageListView(messages: thread.messages)
+                MessageListView(messages: thread.messages.freezeIfNeeded())
```

Comment:
@PhilippeWeidmann I can rollback if this is an issue.

Reply:
@ObservedRealmObject are always frozen so it shouldn't be needed

Reply:
Good to know thanks.